.env

```diff
@@ -71,3 +73,19 @@ CKAN___REDIS_HOST=redis
 CKAN___REDIS_PORT=6379
 CKAN___REDIS_DB=0
 CKAN___REDIS_CONNECT_RETRY=True
+
+## S3 settings
+# The maximum content size, in bytes, for uploads
+CKAN__STORAGE__MAX_CONTENT_LENGTH=650000000
+CKAN_STORAGE_PATH=/var/lib/ckan/files
+CKANEXT__S3SITEMAP__AWS_ACCESS_KEY_ID=_placeholder
+CKANEXT__S3SITEMAP__AWS_BUCKET_NAME=catalog_sitemap
+CKANEXT__S3SITEMAP__AWS_S3_URL=_placeholder
+CKANEXT__S3SITEMAP__AWS_SECRET_ACCESS_KEY=_placeholder
+CKANEXT__S3SITEMAP__AWS_STORAGE_PATH=local
+CKANEXT__S3SITEMAP__REGION_NAME=us-east-1
+CKANEXT__S3SITEMAP__HOST_NAME=http://localstack-container:4566
+CKANEXT__S3SITEMAP__PUBLIC_HOST_NAME=http://localhost:4566
+CKANEXT__S3SITEMAP__LOCALSTACK_ENDPOINT=http://localhost:4566
```

Comment:
Is this still needed, specifically adding the localstack endpoint as a variable?

Reply:
Hmm, good point. I'll investigate a bit and should be able to pull it out.

Reply:
I played around with a couple options for how best to inject the localstack URL, would like to discuss later. Merging for now to check behavior in dev.